sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/implementation/FormRecognizerClientImpl.java

```diff
@@ -103,7 +108,12 @@ public SerializerAdapter getSerializerAdapter() {
         return this.serializerAdapter;
     }
 
-    /** Initializes an instance of FormRecognizerClient client. */
+    /**
+     * Initializes an instance of FormRecognizerClient client.
+     *
+     * @param endpoint Supported Cognitive Services endpoints (protocol and hostname, for example:
```

Comment:
oh interesting. Would it be better to point the user to the service documentation instead of us saying how the endpoint should look like?

Reply:
This is generated code by autorest and swagger.